.python-version

```diff
@@ -1 +0,0 @@
-3.6.3
```

Comment:
@nealedj I can get it to work if this file is back in.
Without this pipenv doesn't know which version of python to use. which to me is quite stupid

Reply:
Hmm, go figure. I've added it back again.